main.py

```diff
@@ -280,14 +283,44 @@ def render_field_link(field):
         name=field['name'],
         id=field_id)
 
+def render_scouting_observation_link(scouting_observation):
+    id = scouting_observation['id']
+    return '<a href="{link}">{id}</a>'.format(
+        link=url_for('scouting_observation', scouting_observation_id=id),
+        id=id)
 
 def redirect_uri():
     """
     :return: Returns uri for redirection after Log In with FieldView.
     """
     return url_for('login_redirect', _external=True)
 
+@app.route('/scouting-observation/<scouting_observation_id>', methods=['GET'])
+def scouting_observation(scouting_observation_id):
+    observation = climate.get_scouting_observation(state('access_token'), CLIMATE_API_KEY, scouting_observation_id)
+   
+    return  """
+            <h1>Partner API Demo Site</h1>
+            <h2>Scouting Observation ID: {scouting_observation_id}</h2>
+            <p>{json}</p>
```

Comment:
Would be good to format the json output nicely. Probably something like:
`<p><code>{json}</code></p>`

and computing json with `json.dump(observation, indent=4)`